llvm/lib/CAS/OnDiskCommon.cpp

```diff
@@ -8,19 +8,41 @@
 
 #include "OnDiskCommon.h"
 #include "llvm/ADT/StringRef.h"
+#include "llvm/Support/CommandLine.h"
 #include "llvm/Support/Error.h"
+#include "llvm/Support/Process.h"
+#include <mutex>
+#include <optional>
 
 using namespace llvm;
 
+cl::opt<uint64_t> OnDiskCASMaxMappingSize(
```

Comment:
Can we use `ManagedStatic` or otherwise avoid the global constructor for this option?

Reply:
Fixed. I guess we don't have a use case for `cl::opt`. Just use a static variable to store the overwritten value.